### DISCUSSION TITLE
what are resources in a REST architecture?

### DISCUSSION THREAD

**Discussion by bibekkunwar** [community/community#43096] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

what are the different resources used in a REST architecture

**Comment by SaileshBK** [comment, marked as answer]
resources are the key architectural element. A resource is any data or service that can be accessed through a network. REST architectures use a uniform interface to access resources, which makes it easy to expose data and services to the web. Some common resources that might be used in a REST architecture include:

Files: Resources can be files such as images, documents, and audio or video clips.
Databases: Resources can be data stored in a database, such as customer information or product inventory.
Services: Resources can be services such as weather reports, stock quotes, or search results.
